shuttle/setup.sh

```diff
@@ -2,7 +2,7 @@
 
 CHAIN_BIN="${CHAIN_BIN:=persistenceCore}"
 CHAIN_DATA_DIR="${CHAIN_DATA_DIR:=.persistenceCore}"
-CHAIN_ID="${CHAIN_ID:=persistencecore-1}"
```

Comment:
any reason for this? test chains only.. was better it didn't coincide with mainnet/testnet ones

Reply:
This won't be used anyways, the chain id is passed from the hermes' config file.